helm/helm.bzl

```diff
@@ -17,6 +17,11 @@ def helm_chart(
         tags = [],
         install_name = None,
         registry_url = None,
+        helm_opts = [],
+        opts = [],
```

Comment:
Can we name this `command_opts` or something? The naming here feels confusing and took me a bit to realize `Additional options to pass to helm.` was saying the options passed before a helm command (`install`, `push`, etc). Bazel's startup args might be a good parallel where they have
```
bazel {startup args} {invocation} {invocation_args}
```
helm has
```
helm {base args} {command} {command args}
```
or something but the naming/docs was hard for me to understand that.